lib/winapi/winmm/Makefile

```diff
@@ -0,0 +1,17 @@
+WINMM_SRCS := \
+	$(wildcard $(NXDK_DIR)/lib/winapi/winmm/*.c)
```

Comment:
What is the reason for choosing a wildcard?

Reply:
I adopted this from another Makefile which still used wildcards.
This should have used explicit paths. I have pushed a fix.